### DISCUSSION TITLE
LDAP Authentication success but can't login

### DISCUSSION THREAD

**Discussion by AhmedAbuElyazeed** [apache/airflow#31280] [Q&A]
Hi Everyone,
After integrating Airflow with ldap, user is authenticated successfully, but the Airflow webserver gives "Invalid login, please try again" message.
The image is from debugging Flask appbuilder and there's a Get 200 so it's successful.

![Capture](https://github.com/apache/airflow/assets/45356521/6b0c4ffd-ec8b-428f-9ae5-c17731ba9ec0)

**Comment by potiuk** [comment]
Talk at FAB discussion groups- they provide authentication for Airflow.
